src/test/scala/nl.knaw.dans.easy.deposit/servlets/UploadSpec.scala

```diff
@@ -185,14 +194,37 @@ class UploadSpec extends TestSupportFixture with ServletFixture with ScalatraSui
     }
   }
 
+  it should "report existing files" in {
+    val bodyParts = createBodyParts(Seq(
+      ("some", "1.txt", "rababera"),
+      ("some", "2.txt", "rababera"),
+      ("some", "3.txt", "rababera"),
+      ("some", "4.txt", "rababera"),
+    ))
+    val uuid = createDeposit
+    val relativeTarget = "some"
+    val absoluteTarget = (testDir / "drafts" / "foo" / uuid.toString / "bag/data" / relativeTarget).createDirectories()
+    (absoluteTarget / "2.txt").createFile()
+    (absoluteTarget / "3.txt").createFile()
+    post(
+      uri = s"/deposit/$uuid/file/$relativeTarget",
+      params = Iterable(),
+      headers = Seq(fooBarBasicAuthHeader),
+      files = bodyParts
+    ) {
+      absoluteTarget.list.size shouldBe 2
+      status shouldBe CONFLICT_409
+      body shouldBe s"Conflict, the following file(s) already exist on the server: some/3.txt, some/2.txt"
```

Comment:
This is slightly different than in #83
Merging somehow went wrong so changing `api.yml` will have to wait for a new PR after merging this PR.

Once the files are staged and the check is performed,
it is hard to track whether it were multiple body parts or a zip.
As a bonus the names of existing files are returned.

Reply:
When checking the merge result I got very confused with #83 and 
https://github.com/janvanmansum/easy-deposit-api/blob/26413829076ee21e3cc1dddb36612932abee7f36/docs/api/api.yml#L646-L650

What happened? 

Aha, the second link belonged to #76, so my merge attempt went well after all.